### ISSUE TITLE
The automated release is failing 🚨

### ISSUE THREAD

**Issue by austince** [chaijs/chai-http#251]
## :rotating_light: The automated release from the `master` branch failed. :rotating_light:

I recommend you give this issue a high priority, so other packages depending on you could benefit from your bug fixes and new features.

You can find below the list of errors reported by **semantic-release**. Each one of them has to be resolved in order to automatically publish your package. I’m sure you can resolve this 💪.

Errors are usually caused by a misconfiguration or an authentication problem. With each error reported below you will find explanation and guidance to help you to resolve it.

Once all the errors are resolved, **semantic-release** will release your package the next time you push a commit to the `master` branch. You can also manually restart the failed CI job that runs **semantic-release**.

If you are not sure how to resolve this, here is some links that can help you:
- [Usage documentation](https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/README.md)
- [Frequently Asked Questions](https://github.com/semantic-release/semantic-release/blob/caribou/docs/support/FAQ.md)
- [Support channels](https://github.com/semantic-release/semantic-release#get-help)

If those don’t help, or if this issue is reporting something you think isn’t right, you can always ask the humans behind **[semantic-release](https://github.com/semantic-release/semantic-release/issues/new)**.

---

### Invalid npm token.

The [npm token](https://github.com/semantic-release/npm/blob/master/README.md#npm-registry-authentication) configured in the `NPM_TOKEN` environment variable must be a valid [token](https://docs.npmjs.com/getting-started/working_with_tokens) allowing to publish to the registry `https://registry.npmjs.org/`.

If you are using Two-Factor Authentication, make configure the `auth-only` [level](https://docs.npmjs.com/getting-started/using-two-factor-authentication#levels-of-authentication) is supported. **semantic-release** cannot publish with the default `auth-and-writes` level.

Please make sure to set the `NPM_TOKEN` environment variable in your CI with the exact value of the npm token.

---

Good luck with your project ✨

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by austince** [comment]
## :rotating_light: The automated release from the `master` branch failed. :rotating_light:

I recommend you give this issue a high priority, so other packages depending on you could benefit from your bug fixes and new features.

You can find below the list of errors reported by **semantic-release**. Each one of them has to be resolved in order to automatically publish your package. I’m sure you can resolve this 💪.

Errors are usually caused by a misconfiguration or an authentication problem. With each error reported below you will find explanation and guidance to help you to resolve it.

Once all the errors are resolved, **semantic-release** will release your package the next time you push a commit to the `master` branch. You can also manually restart the failed CI job that runs **semantic-release**.

If you are not sure how to resolve this, here is some links that can help you:
- [Usage documentation](https://github.com/semantic-release/semantic-release/blob/caribou/docs/usage/README.md)
- [Frequently Asked Questions](https://github.com/semantic-release/semantic-release/blob/caribou/docs/support/FAQ.md)
- [Support channels](https://github.com/semantic-release/semantic-release#get-help)

If those don’t help, or if this issue is reporting something you think isn’t right, you can always ask the humans behind **[semantic-release](https://github.com/semantic-release/semantic-release/issues/new)**.

---

### Invalid npm token.

The [npm token](https://github.com/semantic-release/npm/blob/master/README.md#npm-registry-authentication) configured in the `NPM_TOKEN` environment variable must be a valid [token](https://docs.npmjs.com/getting-started/working_with_tokens) allowing to publish to the registry `https://registry.npmjs.org/`.

If you are using Two-Factor Authentication, make configure the `auth-only` [level](https://docs.npmjs.com/getting-started/using-two-factor-authentication#levels-of-authentication) is supported. **semantic-release** cannot publish with the default `auth-and-writes` level.

Please make sure to set the `NPM_TOKEN` environment variable in your CI with the exact value of the npm token.

---

Good luck with your project ✨

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Comment by austince** [comment]
@keithamus what type of token is `f674......69f0`? semantic-release should now support NPM `automation tokens`, if you're able to make of one those.

**Comment by keithamus** [comment]
That token is an automation token.

**Comment by austince** [comment]
Hmm, if I could go back 2 years and tell myself not to recommend this... I guess I'll look into what else could be misconfigured.

**Comment by austince** [comment]
Any chance you'd be willing to run the release script locally with that token to see if that reproduces the error?

**Comment by austince** [comment]
![Screen Shot 2021-02-26 at 6 03 16 PM](https://user-images.githubusercontent.com/4655775/109364534-ffde1180-785c-11eb-8c37-29f983fc6ca3.png)
@keithamus I can't see the settings for chai-http, but could you double-check what they look like if you get a chance?

**Comment by keithamus** [comment]
They match the settings in your screenshot.

**Comment by austince** [comment]
Have you published anything else with an automation token with a package having these settings?

**Comment by keithamus** [comment]
I've rolled the token. The new one is `ff9b......76c4`. Double checked the user can publish to the package. I'm on mobile for the weekend so can't update the yml, but if you could please update the comment for it and I can approve from mobile. 

Grasping at straws but I've no idea what is really going wrong here 🤷‍♂️

**Comment by austince** [comment]
Yeah I'm very confused as well, I'll update the yml and try it again. If not will raise some issues and see if semantic-release knows what's up.
